quickstarts/maven/spring-boot-helm/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.5.3</version>
+    <version>3.1.3</version>
```

Comment:
Won't bumping quickstart to Spring Boot 3 make quickstart dependent on JDK17?

Reply:
yes, this is also updated in the README.MD